src/content/docs/first-steps/flashing.mdx

```diff
@@ -1,81 +1,57 @@
 ---
 
+import { Aside,Icon ,Tabs, TabItem } from '@astrojs/starlight/components';
 
-import { Tabs, TabItem, Steps, FileTree, Aside } from '@astrojs/starlight/components';
+This page provides step-by-step instructions for flashing the Spotify Car Thing with the Terbium webpage.
+  
+  ### Flashing with Terbium
+    Complete the following steps to flash the Car Thing.
+    
+    1.) Navigate to [thingify.tools](https://thingify.tools/firmware/P3QZbZIDWnp5m_azQFQqP?tab=versions), select the arrow for the latest release, and select the download button to download "version.number.here-thinglabs.zip".
+   
+    2.) Extract the zip into an accessible location.
+   
+    3.) Navigate to [Terbium.app](https://terbium.app/)
 
-Guides lead a user through a specific task they want to accomplish, often with a sequence of steps.
-Writing a good guide requires thinking about what your users are trying to do.
+    4.) Plug in a USB-C-to-C or USB-A-to-C cable into the Car Thing Power Supply USB-C Port.
 
-## Superbird Tool
- 
+    5.) While holding top preset buttons 1 and 4, plug the USB into the computer. This will put the device into USB burning mode.
 
-### Download the tool
-<Steps>
-1. Download the [Superbird Tool](https://github.com/thinglabsoss/superbird-tool) from Thing Labs Github
+        ![Carthing](/src/assets/docs/CarThing.png)
 
-2. Download the latest image **(8.9.2)** from [thingify.tools](https://thingify.tools/firmware/P3QZbZIDWnp5m_azQFQqP?tab=info)
+        <Aside type='caution'>Ensure the USB port power supply and cable are capable of 
+         passing data and providing ample power to the car thing.</Aside>
 
-3. Unzip the zip file into the superbird-tool/images folder
-</Steps>
+    6.) Select the **Done** button. 
 
-<FileTree>
-- superbird-tool/
-  - images/
-    - **8.9.2-thinglabs/**
-</FileTree>
+    ![terbium-done](/src/assets/docs/terbium-done.png)
 
-<br />
+    7.) A new popup should appear with a device listed as "unknown device from Amlogic, Inc". Ensure that an adblock is not stopping this popup. 
+    
+    <Aside type='note'>If no device is identified, make sure the Car Thing is in USB burn mode, see steps 4 and 5.</Aside> 
 
-### Flashing
-<Steps>
-4. Plug the USB end of the Car Thing's USB-C cable into the computer while holding down the top buttons 1 and 4. This will put it into USB burning mode.
-</Steps>
+    8.) Select the device, then **Connect**.
 
-<Aside type='note'>
-  We recommend using a A to C usb 2.0 cable if available, and avoid using hubs or docks to connect your CarThing. 
-</Aside>
+    ![terbium-select-device](/src/assets/docs/terbium-select-device.png)
 
-<br />
-<Steps>
-5. Open the superbird-tool folder on a Terminal window
+    <Aside type='caution'>If the system has been used to flash Car Thing previously using a legacy method
+    an "Access Denied" error may appear. See the Terbium troubleshooting window
+    for details on removing old drivers.</Aside>
```

Comment:
I think there should be explanations of how to fix this sort of issue in the docs.  This also isn't only a driver issue, only a driver issue on Windows.

Reply:
I had mixed feeling on this one. I didnt want to repeat the items outlined in the Terbium webpage at the risk of them getting out of sync.

Reply:
We can just update the wiki if there are sync issues, so it's fine.